examples/dataflow-flex-python/README.md

```diff
@@ -0,0 +1,210 @@
+## Dataflow Python Flex Template
+
+This example contains a sample Dataflow job which reads a XML file and inserts the records to BQ table.
+It explains how to create Flex template and run it in a restricted environment where there is no 
+internet connectivity to dataflow launcher or worker nodes. It also run dataflow template on shared VPC.
+Example also contains a DAG which can be used to trigger Dataflow job from composer. It also demonstrates
+how we can use cloudbuild to implement CI/CD for this dataflow job.
+
+There are two docker files in the repo, the Dockerfile present at the root of directory is to create a image
```

Comment:
I would report main structure of the folder explain what you can find on each subforlder.

Reply:
Added folder structure explanation to README.